ci_example_cpp/include/ci_example_cpp/default_configuration.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "ci_example_cpp/gains_configuration.hpp"
+
+
+#define DEFAULT_KP 1.0
+#define DEFAULT_KD 1.0
+#define DEFAULT_KI 1.0
+
+
+namespace ci_example_cpp {
+
+  /*! Returning default configuration: kp: DEFAULT_KP, kd: DEFAULT_KD, ki: DEFAULT_KI */
+  class Default_configuration : public Gains_configuration {
+    
+  public:
+    /** Here we use the default destructor. */
+    ~Default_configuration(){}
+    
+    double get_kp() const; /*!< Always returns DEFAULT_KP. */
```

Comment:
I understand your point. I am not sure I agree though because:
- 1. It makes the code tidy, specifically on getters/setters/attributes.
- 2. The reviewer will see these changes during the review and would ask case by case no?

Reply:
It is just some small annoyance that could be prevented and I personally don't see any advantage of having it in the same line but I also don't have a strong opinion on this. So I am okay with keeping as it is if you prefer it like that.